rules/cross-platform/persistence_ssh_authorized_keys_modification.toml

```diff
@@ -20,13 +20,13 @@ name = "SSH Authorized Keys File Modification"
 risk_score = 47
 rule_id = "2215b8bd-1759-4ffa-8ab8-55c8e6b32e7f"
 severity = "medium"
-tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Persistence"]
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Lateral Movement", "Persistence"]
 timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
 event.category:file and event.type:(change or creation) and
- file.name:("authorized_keys" or "authorized_keys2") and
+ file.name:("authorized_keys" or "authorized_keys2" or "/etc/ssh/sshd_config" or "/root/.ssh") and
```

Comment:
Recommendation: Add `/opt/jc/bin/jumpcloud-agent` to `not process.executable` list
Reason: Out of ~800 alerts in ~30 days, ~100 related to JumpCloud where they describe automated changes to `/etc/ssh/sshd_config` file being added.
Reference: https://support.jumpcloud.com/s/article/changes-to-the-sshd-configuration-are-reverted-by-the-agent1